diff -r -u littlewizard-1.1.5/include/environment.h littlewizard-1.1.5.new/include/environment.h
--- littlewizard-1.1.5/include/environment.h	2005-09-11 21:17:23.000000000 +0200
+++ littlewizard-1.1.5.new/include/environment.h	2007-09-25 07:16:06.009362586 +0200
@@ -44,7 +44,7 @@
 static xmlNode *initializeXMLTree (xmlDocPtr &doc);	
 static void clear();
 public:
-static void init();
+static void init(GData *the_symbols);
 static void uninit ();
 static LWPixmapSet *getPixmapSet ();
 static xmlNode *getWizardXML ();
diff -r -u littlewizard-1.1.5/liblw/environment.cc littlewizard-1.1.5.new/liblw/environment.cc
--- littlewizard-1.1.5/liblw/environment.cc	2005-11-16 22:25:02.000000000 +0100
+++ littlewizard-1.1.5.new/liblw/environment.cc	2007-09-25 07:19:13.059840366 +0200
@@ -33,8 +33,6 @@
 LWProgram *LWEnvironment::program=NULL;
 gboolean LWEnvironment::enabled_grid=FALSE;
 
-extern GData *get_symbols ();
-
 /*! \brief Tworzy obszar z suwakiem
 
 	Tworzy obszar z suwakiem, modyfikując dodatkowo kolor tła obszaru.
@@ -63,12 +61,13 @@
 
 	Inicjuje środowisko, odczytując ustawienia z pliku
 */
-void LWEnvironment::init ()
+void LWEnvironment::init (GData *the_symbols)
 {
 static gboolean inited=FALSE;
 xmlDoc *doc=NULL;
 	
 	g_assert (inited == FALSE);
+	g_assert (the_symbols != NULL);
 	inited = TRUE;
 	
 	xmlNode *root_node = initializeXMLTree (doc);
@@ -91,7 +90,7 @@
 	icons->restoreFromXML (root_node->children->next->next);
 	gtk_paned_add1 (GTK_PANED (world_paned), icons->getWidget());
 		
-	symbols = get_symbols();
+	symbols = the_symbols;
 	program_paned = gtk_vpaned_new ();
 	g_object_ref (G_OBJECT (program_paned));
 	gtk_widget_show (program_paned);
diff -r -u littlewizard-1.1.5/src/lwtest.cc littlewizard-1.1.5.new/src/lwtest.cc
--- littlewizard-1.1.5/src/lwtest.cc	2006-01-17 20:03:24.000000000 +0100
+++ littlewizard-1.1.5.new/src/lwtest.cc	2007-09-25 07:19:49.570664393 +0200
@@ -15,6 +15,8 @@
 #include "include/support.h"
 #include "include/environment.h"
 
+extern GData *get_symbols();
+
 static void test_save (xmlDoc *doc, char *filename)
 {
 	xmlSaveFile (filename, doc);
@@ -42,7 +44,8 @@
 {
 	gtk_init (&argc, &argv);
 
-	LWEnvironment::init();
+	GData *symbols = get_symbols();
+	LWEnvironment::init(symbols);
 	LWProject *project = new LWProject();
 	LWEnvironment::setProject (project);
 	
diff -r -u littlewizard-1.1.5/src/main.cc littlewizard-1.1.5.new/src/main.cc
--- littlewizard-1.1.5/src/main.cc	2007-07-09 22:24:33.000000000 +0200
+++ littlewizard-1.1.5.new/src/main.cc	2007-09-25 07:50:50.432771577 +0200
@@ -23,7 +23,7 @@
 
 extern void create_interface ();
 extern void set_title (gchar *name);
-
+extern GData *get_symbols ();
 
 int
 main (int argc, char *argv[])
@@ -80,7 +80,8 @@
 	g_list_foreach (icon_list, (GFunc) g_object_unref, NULL);
 	g_list_free (icon_list);
 	
-	LWEnvironment::init();
+	GData *symbols = get_symbols();
+	LWEnvironment::init(symbols);
 	LWProject *project = new LWProject();
 	LWEnvironment::setProject (project);
 
