--- littlewizard-1.2.0rc1/Makefile.am~	2007-11-29 04:25:30.000000000 +0100
+++ littlewizard-1.2.0rc1/Makefile.am	2007-12-16 00:33:49.000000000 +0100
@@ -3,20 +3,11 @@
 SUBDIRS = po liblw liblanguage include src pixmaps data mime
 
 lwdocdir = ${prefix}/doc/littlewizard
-lwdoc_DATA = \
-	README\
-	COPYING\
-	AUTHORS\
-	ChangeLog\
-	INSTALL\
-	COMPATIBILITY\
-	TODO
 
 EXTRA_DIST = $(lwdoc_DATA)
 
 
 gnomemenudir = $(datadir)/applications
-gnomemenu_DATA = littlewizard.desktop
 
 # Copy all the spec files. Of cource, only one is actually used.
 dist-hook:
