--- littlewizard-1.1.5/littlewizard.desktop.in~	2006-09-06 23:01:18.000000000 +0200
+++ littlewizard-1.1.5/littlewizard.desktop.in	2007-09-26 00:27:37.000000000 +0200
@@ -19,9 +19,10 @@
 Comment[sk]=Vývojové prostredie pre deti
 
 Exec=lw
-Icon=@PACKAGE_PIXMAPS_DIR@/littlewizard-icon.png
+Icon=littlewizard-icon.png
 Terminal=false
 MultipleArgs=false
 Type=Application
 Categories=GTK;GNOME;Education;
 Encoding=UTF-8
+# vi: encoding=utf-8
